ambry-api/src/main/java/com/github/ambry/config/HelixAccountServiceConfig.java

```diff
@@ -99,6 +100,20 @@
   @Default("100")
   public final int totalNumberOfVersionToKeep;
 
+  /**
+   * The number of retry times when the update accounts fails by marking delete_in_progress container status to inactive;
+   */
+  @Config(RETRY_COUNT)
+  @Default("10")
+  public final int retryCount;
```

Comment:
nit: can we call this maxRetryCountOnUpdateFailure ?

Reply:
Updated.